github-actions/breaking-changes-label/main.ts

```diff
@@ -0,0 +1,56 @@
+import * as core from '@actions/core';
+import {context} from '@actions/github';
+import {Octokit} from '@octokit/rest';
+import {parseCommitMessage} from '../../ng-dev/commit-message/parse';
+import {getAuthTokenForAngularRobotApp} from '../utils';
+
+const breakingChangesLabel = 'breaking changes';
```

Comment:
Actually it appears that the label in FW has been changed to `flag: breaking change`.

https://github.com/angular/angular/commit/ebc80b3b5c6d05d11093aadeab4c893119ee16cf#diff-0f6460592cf1db69dc570c128942fb78af50c792b862af847654785406727242